editoast/editoast_authz/src/regulator.rs

```diff
@@ -177,7 +206,7 @@ impl<S: StorageDriver> Regulator<S> {
         Ok(())
     }
 
-    /// Removes some users from a group
+    /// Remove some users from a group
```

Comment:
```suggestion
    /// Removes some users from a group
```

🤔 

> What does this function do? It removes X from Y

+ inconsistency with the other docs